packages/generators/add-generator.ts

```diff
@@ -73,7 +72,7 @@ const traverseAndGetProperties = (arr: object[], prop: string): boolean => {
 const searchProps = (answers: object, input: string): Promise<string[]> => {
 	input = input || "";
 	return Promise.resolve(
-		PROPS.filter((prop: string): boolean =>
+		PROPS.filter((prop): boolean =>
```

Comment:
In other occurrences of `.filter` you are keeping the type of the parameter. I would keep here too

Reply:
You are correct here. we need to have it here.